packages/gestalt/src/Datapoint.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow strict
+import React from 'react';
+import { create } from 'react-test-renderer';
+import Datapoint from './Datapoint.js';
+
+describe('Datapoint', () => {
+  it('renders', () => {
+    const tree = create(
+      <Datapoint
+        title="Title"
+        value="1M"
+        trend={{ value: 30, accessibilityLabel: 'Value change icon accessibility label' }}
+      />,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders an accessibility label', () => {
```

Comment:
These tests are identical, so the second one can be deleted

Reply:
Alternatively, can update one of these tests to not have a trend